### PR TITLE
Feature/a11ymap disable button when no edited value

### PR DESCRIPTION
**Pull request by Mayaryin**
The 'send' button in the wheelchair editors is conditionally enabled/disabled if the user changes the current tag value.

I was unsure whether this should be handled by each of the specific editors or the auto editor. The latter seems to be more adequate to avoid code duplication but, in practice, felt like adding more complecxity to the process, since it requires passing back an forth the edited tag value and the button disabled flag between the auto editor and the specific editor.
I would be happy to hear your opinions on this @opyh @kriskbx

### CODE REVIEW

File: src/components/CombinedFeaturePanel/editors/StringFieldEditor.tsx [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+  onChange,
+  handleSubmitButtonClick,
 }: BaseEditorProps) => {
-  const { baseFeatureUrl } = useContext(FeaturePanelContext)
-  const current = feature.properties?.[tagKey] || ''
-  const [value, setValue] = React.useState(current)
+  const { baseFeatureUrl } = useContext(FeaturePanelContext);
+  const current = feature.properties?.[tagKey] || "";
+  const [editedTagValue, setEditedTagValue] = React.useState(current);
+  const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(true);
```
Comment (Mayaryin) [inline src/components/CombinedFeaturePanel/editors/StringFieldEditor.tsx:20]: This is new

File: src/components/CombinedFeaturePanel/editors/StringFieldEditor.tsx [hunk 2]
```diff
@@ -27,20 +34,24 @@ export const StringFieldEditor = ({
...
 
       <footer className="_footer">
         <AppStateLink href={baseFeatureUrl}>
-          <div role="button" className="_option _back">Back</div>
+          <div role="button" className="_option _back">
+            Back
+          </div>
         </AppStateLink>
-        <Button onClick={handleSubmitButtonClick}>Send</Button>
+        <Button onClick={handleSubmitButtonClick} disabled={isButtonDisabled}>
```
Comment (Mayaryin) [inline src/components/CombinedFeaturePanel/editors/StringFieldEditor.tsx:51]: And here

Comment (kriskbx) [reply src/components/CombinedFeaturePanel/editors/StringFieldEditor.tsx:51]: I agree with @opyh. Great idea 🙌

File: src/components/CombinedFeaturePanel/editors/AutoEditor.tsx [hunk 3]
```diff
@@ -132,22 +144,24 @@ export const AutoEditor = ({
...
-        {feature['@type'] === 'osm:Feature' && (
+        {feature["@type"] === "osm:Feature" && (
           <FeatureImage feature={feature} />
         )}
       </FeatureNameHeader>
       <h2 className="_title">{t`No editor available for ${tagKey}`}</h2>
       <footer className="_footer">
         <AppStateLink href={baseFeatureUrl}>
-          <div role="button" className="_option _back">Back</div>
+          <div role="button" className="_option _back">
```
Comment (opyh) [inline src/components/CombinedFeaturePanel/editors/AutoEditor.tsx:160]: Could this be a Radix `<Button>` instead? :) Or later?

**Review by opyh — APPROVED** [review]
Thanks! Looks good and works as expected! 💯

Just a few nitpicks. If you think they make sense, feel free to make the changes – and to rebase & merge yourself.

**Pull request merged**